core/primitives/src/lib.rs

```diff
@@ -83,19 +83,19 @@ pub const HOURS: BlockNumber = MINUTES * 60;
 pub const DAYS: BlockNumber = HOURS * 24;
 
 // Unit = the base number of indivisible units for balances
-/// 1e18 wei — 1,000,000,000,000,000,000
+/// 1e18 WEI — 1,000,000,000,000,000,000
```

Comment:
I suggest keeping the original style.

These are copied items.

Reply:
Where did you copy it from?